mysql-test/suite/rpl/t/rpl_bug-lp-1667307.test

```diff
@@ -0,0 +1,51 @@
+#
+# In the case if the bug is not fixed the test will crash on assert
+# in Rows_query_log_event::do_apply_event().
+#
+--source include/have_debug.inc
```

Comment:
Laurynas, I just want to clarify about regression testing. My understanding of regression supposes that there is the criteria of regression. And this is not only my understading. https://en.wikipedia.org/wiki/Regression_testing : "Regression testing is a type of software testing which verifies that software which was previously developed and tested still performs correctly after it was changed or interfaced with other software." What means the criteria of correctness must exist. In our case such criteria exists only in debug build. Even if something is broken in release build we will not see this in the current version of the test. In this test it is supposed that some code (assertion) must be executed to specify such criteria, what happens only in debug build. So what the sense of this test in release build if there is no way to cach regression for release build?